src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/ModuleScaffold.kt

```diff
@@ -75,6 +82,52 @@ object ModuleScaffold : Module("Scaffold", Category.WORLD) {
         val failedAttemptsOnly by boolean("FailedAttemptsOnly", true)
     }
 
+    object AutoJump : ToggleableConfigurable(this, "AutoJump", false) {
```

Comment:
Done, But should we maybe rename all of them to just the name? so instead of `ScaffoldAutoJumpFeature.kt` it would just be `AutoJump.kt`. the current names just seem really long, and it is already indicated what they are for by the position of the files in the folders.

Reply:
I also was thinking about a better solution when refactoring. But I think that those long names are easier to find with class search, because "scaffold stablize movement" will take you directly to the feature. Additionally, you won't confuse `scaffold/Eagle.kt` with `ModuleEagle.kt`. We only use those names a few times so the trade-off is worth it